src/rust/grapl-web-ui/tests/api/plugin/create.rs

```diff
@@ -0,0 +1,76 @@
+use grapl_web_ui::routes::api::plugin::{
+    create::CreateResponse,
+    get_metadata::GetPluginMetadataResponse,
+};
+
+#[actix_web::test]
+async fn plugin_create() -> eyre::Result<()> {
+    let app = crate::test_app::TestApp::init().await?;
+
+    app.login_with_test_user().await?;
+
+    let plugin_name = "Test Plugin";
+    let create_metadata_body = serde_json::json!({
+            "plugin_name": plugin_name,
+            "plugin_type": "generator",
+            "event_source_id": "00000000-0000-0000-0000-000000000000"
```

Comment:
add a TODO?

Reply:
i mean, you could make a real event source with the event source client

Reply:
Sounds to me more like something for an e2e test. wdyt?

Reply:
that line keeps blurring :) i worry about the hardcoded event source id since, well, couldn't somebody else start using that and corrupting this test?

Reply:
Kk, I'm generating a random Uuid instead.

I don't want to use the event source client, that sounds like overkill for this integration test.